src/test/PoolModifyPositionTest.sol

```diff
@@ -53,7 +53,7 @@ contract PoolModifyPositionTest is Test, PoolTestBase {
         (,,, int256 delta1) = _fetchBalances(data.key.currency1, data.sender);
 
         // These assertions only apply in non lock-accessing pools.
-        if (!data.key.hooks.hasPermissionToAccessLock()) {
+        if (!data.key.hooks.hasPermissionToAccessLock() && !data.key.fee.hasHookWithdrawFee()) {
```

Comment:
Why does this change in this PR? Are we hitting a fuzz case here?

Reply:
Yeah fuzz case - a hook withdrawal fee changes the output amounts expected of a modify position.
This addition will be removed again in #432